_sass/_header.scss

```diff
@@ -20,7 +20,7 @@ header {
 
   .logo {
     background-color: none;
-    background-image: url("images/favicon2.png");
+    background-image: url("images/logo.png");
```

Comment:
I like the logo change idea, but the logo here is awful quality and pixelated
<img width="1100" alt="Screenshot 2019-10-06 at 19 28 18" src="https://user-images.githubusercontent.com/7111514/66272906-71d04800-e86f-11e9-81bc-afab06626c26.png">

Reply:
do you have an svg or some high quality version to start from? Because I had to download it from facebook and convert it with imagemagick and I guess it's not the best way.

Reply:
When we figure this out we should upload our file to this repository https://github.com/techworkersco/shared_resources

and here is a higher resolution file.  

![LTWC_logo_black_red_berlin_tagline](https://user-images.githubusercontent.com/7111514/66277330-f9807b80-e89c-11e9-90e8-37186e9f4523.png)

Reply:
And the SVG source code (without bottom text)

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   version="1.1"
   id="svg29"
   xml:space="preserve"
   width="480"
   height="480"
   viewBox="0 0 480 480"
   sodipodi:docname="LTWC_logo_black_red_berlin_tagline.svg"
   inkscape:version="0.92.2 5c3e80d, 2017-08-06"
   inkscape:export-filename="/Users/tomjennings/Documents/projects/TWC/LTWC_logo_black_red_berlin_tagline.png"
   inkscape:export-xdpi="959.94226"
   inkscape:export-ydpi="959.94226"><metadata
     id="metadata35"><rdf:RDF><cc:Work
         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
     id="defs33" /><sodipodi:namedview
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1"
     objecttolerance="10"
     gridtolerance="10"
     guidetolerance="10"
     inkscape:pageopacity="1"
     inkscape:pageshadow="2"
     inkscape:window-width="1276"
     inkscape:window-height="755"
     id="namedview31"
     showgrid="true"
     showguides="false"
     inkscape:guide-bbox="true"
     inkscape:zoom="1.8684821"
     inkscape:cx="240.05642"
     inkscape:cy="213.28713"
     inkscape:window-x="4"
     inkscape:window-y="1"
     inkscape:window-maximized="1"
     inkscape:current-layer="g854"
     inkscape:snap-grids="false"><sodipodi:guide
       position="239.84227,240.46849"
       orientation="1,0"
       id="guide4750"
       inkscape:locked="false" /><inkscape:grid
       type="xygrid"
       id="grid4761" /><sodipodi:guide
       position="0,0"
       orientation="0,480"
       id="guide1187"
       inkscape:locked="false" /><sodipodi:guide
       position="480,0"
       orientation="-480,0"
       id="guide1189"
       inkscape:locked="false" /><sodipodi:guide
       position="480,480"
       orientation="0,-480"
       id="guide1191"
       inkscape:locked="false" /><sodipodi:guide
       position="0,480"
       orientation="480,0"
       id="guide1193"
       inkscape:locked="false" /></sodipodi:namedview><g
     id="g37"
     inkscape:groupmode="layer"
     inkscape:label="TWC_logo_final"
     transform="matrix(1.3333333,0,0,-1.3333333,0,480)"
     style="display:inline"
     sodipodi:insensitive="true"><g
       id="g1158"
       transform="translate(121.7817,199.4143)"><path
         d="m 0,0 v -2.592 c 0,-2.304 -1.008,-3.601 -3.6,-3.601 -2.591,0 -3.601,1.297 -3.601,3.601 v 9.793 h 35.572 v -9.793 c 0,-2.304 -1.008,-3.601 -3.601,-3.601 -2.592,0 -3.6,1.297 -3.6,3.601 V 0 h -6.984 v -92.392 c 0,-2.304 -1.008,-0.599 -3.6,-0.599 -2.594,0 -3.602,1.295 -3.602,3.599 L 6.984,0 Z"
         style="fill:#ec2227;fill-opacity:1;fill-rule:nonzero;stroke:none"
         id="path1160"
         inkscape:connector-curvature="0" /></g><g
       id="g1162"
       transform="translate(175.686,150.594)"><path
         d="m 0,0 h -5.4 c -2.522,0 -3.168,0.432 -4.393,2.592 l -2.52,4.392 c -0.937,1.584 -1.009,1.729 -1.009,3.672 v 99.354 c 0,2.304 1.009,3.601 3.601,3.601 2.592,0 3.6,-1.297 3.6,-3.601 V 10.656 l 1.943,-3.455 h 3.098 l 1.943,3.455 v 22.682 c 0,2.232 1.008,4.25 3.602,4.25 2.592,0 3.599,-2.018 3.599,-4.25 V 10.656 l 1.944,-3.455 h 3.096 l 1.945,3.455 V 52.42 c 0,2.305 1.008,3.601 3.599,3.601 2.593,0 3.602,-1.296 3.602,-3.601 V 10.656 c 0,-1.943 -0.072,-2.088 -1.009,-3.672 L 18.721,2.592 C 17.497,0.432 16.849,0 14.328,0 h -5.4"
         style="fill:#231f20;fill-opacity:1;fill-rule:nonzero;stroke:none"
         id="path1164"
         inkscape:connector-curvature="0" /></g><g
       id="g1166"
       transform="translate(227.7309,150.594)"><path
         d="m 0,0 c -5.4,0 -8.496,2.16 -10.801,6.84 l -6.625,13.394 c -1.223,2.448 -1.943,4.752 -1.943,7.561 0,3.168 0.576,5.111 2.015,7.992 l 6.625,13.32 c 2.448,4.969 5.544,6.914 11.088,6.914 h 12.243 c 1.584,0 3.599,-0.865 3.599,-3.601 0,-2.592 -1.943,-3.6 -3.599,-3.6 H 0.576 c -3.096,0 -3.6,-0.289 -4.969,-3.025 l -6.263,-12.527 c -0.937,-1.946 -1.513,-3.243 -1.513,-5.473 0,-2.018 0.648,-3.24 1.513,-4.969 l 6.12,-12.457 c 1.44,-2.953 2.16,-3.168 5.4,-3.168 h 30.685 c 1.584,0 3.599,-0.865 3.599,-3.601 0,-2.592 -1.943,-3.6 -3.599,-3.6 z"
         style="fill:#ec2227;fill-opacity:1;fill-rule:nonzero;stroke:none"
         id="path1168"
         inkscape:connector-curvature="0" /></g><g
       id="g1170"
       transform="matrix(1.010084,0,0,1.010084,180.00051,262.92942)"><path
         d="m 0,0 c -45.271,0 -82.102,-36.831 -82.102,-82.102 0,-45.271 36.831,-82.101 82.102,-82.101 45.271,0 82.101,36.83 82.101,82.101 C 82.101,-36.831 45.271,0 0,0 m 0,-171.203 c -49.131,0 -89.102,39.971 -89.102,89.101 C -89.102,-32.971 -49.131,7 0,7 49.13,7 89.101,-32.971 89.101,-82.102 89.101,-131.232 49.13,-171.203 0,-171.203"
         style="fill:#231f20;fill-opacity:1;fill-rule:nonzero;stroke:none"
         id="path1172"
         inkscape:connector-curvature="0" /></g></g><g
     inkscape:groupmode="layer"
     id="layer1"
     inkscape:label="Text"
     style="display:inline"><path
       transform="rotate(6.9143347)"
       d="m 397.147,209.36209 a 130,130 0 0 1 -129.98241,130 A 130,130 0 0 1 137.14701,209.39726 130,130 0 0 1 267.09425,79.362102 130,130 0 0 1 397.14698,209.29176 l -129.99998,0.0703 z"
       sodipodi:end="6.2826443"
       sodipodi:start="0"
       sodipodi:ry="130"
       sodipodi:rx="130"
       sodipodi:cy="209.36209"
       sodipodi:cx="267.147"
       sodipodi:type="arc"
       id="path5017"
       style="display:inline;opacity:0.49000005;fill:none;stroke:none;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" /><g
       id="g854"
       transform="matrix(1.0313143,0.06911043,-0.06911043,1.0313143,14.573199,-28.914993)"><text
         y="-1022.4825"
         x="-1452.0664"
         xml:space="preserve"
         style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;opacity:1;fill:#e6141b;fill-opacity:1;stroke:none;stroke-width:0.75"
         id="text1326"
         inkscape:transform-center-x="-0.9091119"
         inkscape:transform-center-y="81.68682"
         transform="rotate(0.58086001,308.16282,267.03754)"><textPath
           xlink:href="#path845"
           id="path1315"><tspan
   style="font-size:24px;fill:#e6141b;fill-opacity:1"
   id="tspan1322"><tspan
   id="tspan1318"
   style="fill:#000000">BERLIN</tspan> TECH <tspan
   id="tspan1320"
   style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-family:sans-serif;-inkscape-font-specification:'sans-serif Bold';fill:#000000;fill-opacity:1">WORKERS</tspan> COALITION</tspan></textPath></text>
<path
         style="display:inline;opacity:0.49000005;fill:none;stroke:none;stroke-width:1.1153847;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
         id="path845"
         sodipodi:type="arc"
         sodipodi:cx="-203.79654"
         sodipodi:cy="271.50873"
         sodipodi:rx="145"
         sodipodi:ry="145"
         sodipodi:start="0"
         sodipodi:end="6.2826443"
         d="m -58.796539,271.50873 a 145,145 0 0 1 -144.980391,145 145,145 0 0 1 -145.0196,-144.96078 145,145 0 0 1 144.94116,-145.03921 145,145 0 0 1 145.058809,144.92154 l -144.999979,0.0785 z"
         transform="matrix(-0.99272725,0.12038521,0.12038521,0.99272725,0,0)" /></g></g><g
     style="display:inline"
     inkscape:label="Tagline"
     id="g974"
     inkscape:groupmode="layer"><text
       inkscape:transform-center-y="-75.147272"
       inkscape:transform-center-x="-1.3001385"
       id="text966"
       xml:space="preserve"
       transform="rotate(178.13492,239.94112,239.79792)"><textPath
         xlink:href="#path1307"
         id="textPath1312"
         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:24px;line-height:7.25;font-family:Courier;-inkscape-font-specification:Courier;font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:-1px;writing-mode:lr-tb;text-anchor:start;opacity:1;fill:#000000"
         dy="-2.5999994">Tech Won't Save Us: Organise!</textPath></text>
<flowRoot
       xml:space="preserve"
       id="flowRoot4665"
       style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"><flowRegion
         id="flowRegion4667"><rect
           id="rect4669"
           width="5"
           height="45"
           x="290"
           y="235" /></flowRegion><flowPara
         id="flowPara4671" /></flowRoot><path
       style="opacity:0.49000005;fill:none;stroke:none;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
       id="path1307"
       sodipodi:type="arc"
       sodipodi:cx="267.147"
       sodipodi:cy="209.36209"
       sodipodi:rx="130"
       sodipodi:ry="130"
       sodipodi:start="0"
       sodipodi:end="6.2826443"
       d="m 397.147,209.36209 a 130,130 0 0 1 -129.98241,130 A 130,130 0 0 1 137.14701,209.39726 130,130 0 0 1 267.09425,79.362102 130,130 0 0 1 397.14698,209.29176 l -129.99998,0.0703 z"
       transform="rotate(6.9143347)" /></g></svg>
```

Reply:
in the end the old logo just looks better, it's too small to include the writings. I committed the new logo anyway to be used in case we redesign the homepage